src/test/ui/rfc-2294-if-let-guard/feature-gate.stderr

```diff
@@ -187,134 +187,6 @@ LL |     use_expr!((let 0 = 1));
    = help: add `#![feature(let_chains)]` to the crate attributes to enable
    = help: you can write `matches!(<expr>, <pattern>)` instead of `let <pattern> = <expr>`
 
-error: `let` expressions are not supported here
-  --> $DIR/feature-gate.rs:10:16
-   |
-LL |         () if (let 0 = 1) => {}
-   |                ^^^^^^^^^
-   |
-   = note: only supported directly without parentheses in conditions of `if`- and `while`-expressions, as well as in `let` chains within parentheses
```

Comment:
Centril also ignored theses tests https://github.com/rust-lang/rust/pull/68577/files#diff-95f7fe3af3dc85b9699c156fbdd6afacadeb79bc450a1890481110d621d15d01. Not sure if they should be restored.